src/components/ChatLog.js

```diff
@@ -0,0 +1,47 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import {DateTime} from 'luxon';
+
+
+const ChatLog = (props) => {
+    const chatComponents = entries => (entries.map(entry => {
+        return (
+            <li key={entry.id}>
+                <ChatEntry
+                id={entry.id}
+                sender={entry.sender}
+                body={entry.body}
+                timeStamp={entry.timeStamp}
+                liked={entry.liked}
+                updateHeart={props.updateHeart}
+                />
+            </li>
+        );
+    }));
+
+    return (
+        <section>
+            <h2>Chat Log</h2>
+            <ul>
+                {chatComponents(props.entries)}
+            </ul>
+        </section>
+    );
+};
+
+
+ChatLog.propTypes = {
+    messages: PropTypes.arrayOf(PropTypes.shape({
+        id: Number.isRequired,
+        sender: String.isRequired,
+        body: String.isRequired, 
+        timeStamp: DateTime.isRequired,
```

Comment:
I don't think these constructors have a `isRequired` field. For prop types, you want to use things like `PropTypes.number.isRequired`. Also, the `timeStamp` is probably a string!